### PR TITLE
[RHCLOUD-23629] Update Cost Management templates

### PR DESCRIPTION
**Pull request by g-duval**
image

image

image

image

image

image

image

### CODE REVIEW

File: engine/src/main/java/com/redhat/cloud/notifications/templates/TemplateService.java [hunk 1]
```diff
@@ -25,6 +25,8 @@
 @ApplicationScoped
 public class TemplateService {
 
+    public static final String HCC_LOGO_TARGET = "Logo-Red_Hat-Hybrid_Cloud_Console-A-Reverse-RGB";
```
Comment (josejulio) [inline engine/src/main/java/com/redhat/cloud/notifications/templates/TemplateService.java:28]: Same as the others - lets move this to tests.

**Review by josejulio — CHANGES_REQUESTED** [review]
Great work!

I just have the nitpick about the variable being used only by tests

File: engine/src/main/java/com/redhat/cloud/notifications/templates/TemplateService.java (hunk 1 above)
Comment (g-duval) [reply engine/src/main/java/com/redhat/cloud/notifications/templates/TemplateService.java]: Sure, I did it

**Review by josejulio — APPROVED** [review]
lgtm

**Pull request merged**